Mail/Views/Thread/Message/Calendar/CalendarBodyDetailsView.swift

```diff
@@ -44,12 +44,23 @@ struct CalendarLabelStyle: LabelStyle {
 }
 
 struct CalendarBodyDetailsView: View {
+    @EnvironmentObject private var mailboxManager: MailboxManager
+
     let event: CalendarEvent
-    let attachmentMethod: AttachmentEventMethod?
-    let iAmInvited: Bool
+
+    private var me: Attendee? {
+        return event.getMyAttendee(currentMailboxEmail: mailboxManager.mailbox.email)
```

Comment:
I checked in your branch, `Attendee` is persisted in realm.
With @PhilippeWeidmann  we try to make it clear in the code when something is `frozen` or should be frozen.

Therefore, `getMyAttendee` could be `getMyFrozenAttendee` and `me` could be `frozenMe`.
If you need a live object instead, it could be `live` instead of `frozen`.